organizing_hub/forms.py

```diff
@@ -141,6 +175,59 @@ def clean_new_password2(self):
         return password2
 
 
+class PasswordResetRequestForm(AuthPasswordResetRequestForm):
+    email = forms.EmailField(
+        label=_("Email Address"),
+        max_length=254
+    )
+
+    def get_users(self, email):
```

Comment:
obviously out of scope, but seems like we should not be creating new user objects in a get method. maybe add a todo or follow up ticket?